core/src/main/scala/org/apache/spark/internal/io/SparkHadoopMapReduceWriter.scala

```diff
@@ -119,7 +119,7 @@ object SparkHadoopMapReduceWriter extends Logging {
     }
   }
 
-  /** Write a RDD partition out in a single Spark task. */
+  /** Write an RDD partition out in a single Spark task. */
```

Comment:
We should replace them to `an RDD`. 

I think I found and replace them by

```bash
find . -type f -name "*.scala"  | xargs sed -i '' 's/ A RDD / An RDD /g'
find . -type f -name "*.scala"  | xargs sed -i '' 's/ A RDD\./ An RDD\./g'
find . -type f -name "*.scala"  | xargs sed -i '' 's/ a RDD / an RDD /g'
find . -type f -name "*.scala"  | xargs sed -i '' 's/ a RDD\./ an RDD\./g'
find . -type f -name "*.java"  | xargs sed -i '' 's/ A RDD / An RDD /g'
find . -type f -name "*.java"  | xargs sed -i '' 's/ A RDD\./ An RDD\./g'
find . -type f -name "*.java"  | xargs sed -i '' 's/ a RDD / an RDD /g'
find . -type f -name "*.java"  | xargs sed -i '' 's/ a RDD\./ an RDD\./g'
find . -type f -name "*.r"  | xargs sed -i '' 's/ A RDD / An RDD /g'
find . -type f -name "*.r"  | xargs sed -i '' 's/ A RDD\./ An RDD\./g'
find . -type f -name "*.r"  | xargs sed -i '' 's/ a RDD / an RDD /g'
find . -type f -name "*.r"  | xargs sed -i '' 's/ a RDD\./ an RDD\./g'
find . -type f -name "*.py"  | xargs sed -i '' 's/ A RDD / An RDD /g'
find . -type f -name "*.py"  | xargs sed -i '' 's/ A RDD\./ An RDD\./g'
find . -type f -name "*.py"  | xargs sed -i '' 's/ a RDD / an RDD /g'
find . -type f -name "*.py"  | xargs sed -i '' 's/ a RDD\./ an RDD\./g'
find . -type f -name "*.md"  | xargs sed -i '' 's/ A RDD / An RDD /g'
find . -type f -name "*.md"  | xargs sed -i '' 's/ A RDD\./ An RDD\./g'
find . -type f -name "*.md"  | xargs sed -i '' 's/ a RDD / an RDD /g'
find . -type f -name "*.md"  | xargs sed -i '' 's/ a RDD\./ an RDD\./g'
```

If there are more, we should fix. Actually, i was preparing a minor PR to fix all of them including `a X..` to `an X..`.